### PR TITLE
Add Conway's Game of Life in Java

### PR DESCRIPTION
**Pull request by vshelke**
Added implementation in Java using Swing Toolkit.

### CODE REVIEW

File: code/unclassified/conways_game_of_life/Conway.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+		cp.setLayout(new BorderLayout());
+		cp.add(gd, BorderLayout.CENTER);
+		cp.add(st_bar, BorderLayout.SOUTH);
+
+		setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
+		setTitle("Game of Life!");
+		setLocationRelativeTo(null);
+		setSize(600, 600);
+		setVisible(true);
+			try {
```
Comment (arnavb) [inline code/unclassified/conways_game_of_life/Conway.java]: unindent this line

**Comment by AdiChat** [comment]
Looks good 👍 

Hope you are enjoying your journey with **OpenGenus**

**Comment by vshelke** [comment]
Thank you very much @AdiChat .
Yes, its amazing 😊 .

**Comment by AdiChat** [comment]
Glad to hear 😍

**Pull request merged**